layouts/shortcodes/get-started-note.html

```diff
@@ -1,6 +1,6 @@
 <div class="note note-info" role="alert">
     <p>
-        <span>Get Started.</span> Pulumi supports many clouds using the same languages, CLI, and deployment
+        <span>GET STARTED:</span> Pulumi supports many clouds using the same languages, CLI, and deployment
```

Comment:
Also here. I'd prefer make these these lower/titlecase. (The additional weight is enough to make them stand out.)